python/pyenv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    pyenv
+version                 2.3.2
+categories-append       python devel
+license                 MIT
+maintainers             nomaintainer
+supported_archs         noarch
+description             Simple Python Version Management
+long_description        pyenv lets you easily switch between multiple versions of Python. It's simple, unobtrusive, and follows the UNIX tradition of single-purpose tools that do one thing well.
+homepage                https://github.com/pyenv/pyenv
+master_sites            https://github.com/pyenv/pyenv/archive/refs/tags/v${version}.tar.gz
```

Comment:
`homepage` and `master_sites` will be set automagically from the `github.setup` line